test/integration/targets/setup_flatpak_remote/tasks/main.yaml

```diff
@@ -0,0 +1,21 @@
+- name: Set up dummy flatpak repository remote
+  block:
+
+    - name: Copy repo into place
+      unarchive:
+        src: repo.tar.xz
+        dest: "{{ remote_tmp_dir }}"
+        owner: root
+        group: root
+        mode: 0644
+
+    - name: Create deterministic link to temp directory
+      file:
+        state: link
+        src: "{{ remote_tmp_dir }}/"
+        path: "/tmp/flatpak"
+        owner: root
+        group: root
+        mode: 0644
+
+  become: true
```

Comment:
Include a handler that removes the link at the end of the test. Take a look at setup_remote_tmp_dir for an example:

https://github.com/ansible/ansible/tree/devel/test/integration/targets/setup_remote_tmp_dir/

The temp dir itself will be cleaned up by the handler from setup_remote_tmp_dir.

Reply:
Sorry for the delay! Thank you for the feedback, I implemented this in my latest commit.